go-selinux/selinux_linux.go

```diff
@@ -781,6 +781,20 @@ func selinuxEnforcePath() string {
 	return filepath.Join(getSelinuxMountPoint(), "enforce")
 }
 
+func selinuxMLSEnabledPath() string {
+	return filepath.Join(getSelinuxMountPoint(), "mls")
+}
+
+// isMLSEnabled checks if MLS is enabled.
+func isMLSEnabled() bool {
+	enabledB, err := os.ReadFile(selinuxMLSEnabledPath())
```

Comment:
I see it's done in other places as well, but I think the `selinuxMLSEnabledPath()` may be a bit "too much abstraction", especially as  we only use it in a single place; perhaps just inline it;

```go
	enabledB, err := os.ReadFile(filepath.Join(getSelinuxMountPoint(), "mls"))
```